app/models/spotlight/solr_harvester.rb

```diff
@@ -75,9 +75,10 @@ def harvest_item(record, job_tracker, job_progress)
       handle_item_harvest_error(e, parsed_solr_item, job_tracker)
     end
 
-    def solr_harvests(page = nil)
-      page = page.presence || 1
-      solr_connection.paginate(page, ROW_COUNT, 'select', params: { q: '*:*', wt: 'json' })
+    def solr_harvests(cursor = nil)
+      cursor = cursor.presence || '*'
+
+      solr_connection.get('select', params: { q: '*:*', cursorMark: cursor, sort: '_id asc', rows: ROW_COUNT, wt: 'json' })
```

Comment:
Is `_id asc` the correct way to sort the incoming results? `_id` seemed to be the only uniquely identifying field in the data I saw. 

cc @phil-plencner-hl @dl-maura @VanessaVenti

Reply:
That sounds right to me, unless Phil and Maura have other thoughts.

Reply:
ID seems like the best bet.

Reply:
See related commit: https://github.com/harvard-lts/spotlight-oaipmh-resources/pull/43/commits/9ea83ac1490c45f2d6ff5a62d08ab8a40cce3ceb